VERSION

```diff
@@ -1 +1 @@
-0.44.10
\ No newline at end of file
+0.45.0
```

Comment:
0.45.1 includes the CCCS changes https://github.com/cds-snc/notification-manifests/releases/tag/v0.45.1